WordleGame/WordleGame/Models/Game/GameManager.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  GameManager.swift
+//  WordleGame
+//
+//  Created by Alex on 3.05.22.
+//
+
+import Foundation
+import UIKit
+
+struct GameManager {
+    
+    private var currentLetterIndexInRow = 0
+    private var currentAttemptIndex = 0
+    
+    private let resultWord: String
+    
+    var countOfLetters: Int = 0
+    var countOfAttempts: Int = 0
+    
+    var gameField: [[LetterBox?]]
+    
+    init() {
+        self.resultWord = getRandomWordFromTXTFile()
+        self.countOfLetters = resultWord.count
+        self.countOfAttempts = resultWord.count
```

Comment:
Why do you set `countOfAttempts` to the number of letters in a word?

Reply:
I used countOfAttempts to the number of letters in a word because I want create next algorithm:
If we get random word less than 5 symbols, we set 5 attempts. If we get random word more than 5 symbols, we set attempts to the number of letters. I will realise this algorithm when I add txt file with more words, not with 5 symbols words.